safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py

```diff
@@ -296,8 +329,19 @@ def run(self):
             number_of_fatalities[mmi] = fatalities
 
         # Total statistics
+        total_fatalities_raw = numpy.nansum(
+            number_of_fatalities.values(), axis=0)
+
+        # Compute probaility of fatality in each magnitude bin
+        if (self.__class__.__name__ == 'PAGFatalityFunction') or (
```

Comment:
Hmm this is vulnerable to name changes in the class....I guess our Unit tests will catch breakages, but I wonder if there is another approach we can take...